src/oic/__init__.py

```diff
@@ -3,9 +3,22 @@
 
 # Since SystemRandom is not available on all systems
 try:
-    import random.SystemRandom as rnd
+    # Python 3.6+, designed for this usecase
+    from secrets import choice
```

Comment:
I believe that even this should be initialized with a call to `SystemRandom`.

Reply:
Not needed. The secrets module does that under the hood.

See 
https://github.com/python/cpython/blob/6ea20fc719dcaf102d2cefa1786f0421514f7a58/Lib/secrets.py#L22

Reply:
OK.